docs/user-manual/en/configuring-transports.md

```diff
@@ -403,6 +403,15 @@ following additional properties:
     This property is only for an `acceptor`. It tells a client
     connecting to this acceptor that 2-way SSL is required. Valid values
     are `true` or `false`. Default is `false`.
+    
+-   `wantClientAuth`
+
+    This property is only for an `acceptor`. It tells a client
+    connecting to this acceptor that 2-way SSL is requested but not required. 
+    Valid values are `true` or `false`. Default is `false`. 
+    
+    Note that this setting will override `needClientAuth` if both properties 
+    are set to true.
 
```

Comment:
I failed to read this before my previous comment but as before I'd have opted to go the other way on the choice and defaulted to need if both are there.  For reference the 5.x code makes that choice as well when both are present on the URI